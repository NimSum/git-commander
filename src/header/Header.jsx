import React, { Component } from 'react'

export function Header () {
    return (
      <header>
        <h1>GIT COMMANDER</h1>
      </header>
    )
}

export default  Header;
