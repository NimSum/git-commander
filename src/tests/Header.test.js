import React from 'react';
import Header from '../header/Header';
import Adapter from 'enzyme-adapter-react-16';
import { shallow , configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('Header' , () => {
  
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(
      < Header />
    );
  });

  it('Should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })


})
