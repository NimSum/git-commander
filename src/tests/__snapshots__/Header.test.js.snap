// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should match snapshot 1`] = `
<header>
  <h1>
    GIT COMMANDER
  </h1>
</header>
`;
